frontend/src/pages/arrange_components/arrange_components.js

```diff
@@ -1,27 +1,16 @@
-import React from 'react'
-
-import "react-grid-layout/css/styles.css";
-import "../../../node_modules/react-resizable/css/styles.css";
-import { Responsive, WidthProvider } from 'react-grid-layout';
-import VisualComponentsLayout from "./visual_components_layout";
-import styled from "styled-components";
-import axios from "axios";
-import {confirmAlert} from "react-confirm-alert";
-import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
-import {faQuestion} from "@fortawesome/free-solid-svg-icons";
-import "../../pages.css"
-require('dotenv').config();
+import '../../../node_modules/react-resizable/css/styles.css';
+import '../../pages.css';
+import 'react-grid-layout/css/styles.css';
+
+import { faQuestion } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import React from 'react';
+import { confirmAlert } from 'react-confirm-alert';
```

Comment:
Is there a reason not to follow this import order?

1. stdlib imports;
2. 3rd party imports;
3.  local/library specific imports

Reply:
I didn't do it on purpose. I use an extension that sorts that automatically.
1. css files
2. everything which is located in node_modules folder
3. local specific imports

If you like, I can change it :)